src/index.html

```diff
@@ -1,13 +1,508 @@
+              </svg>
+            </button>
+            <div class="menu top-bar__menu">
+              <div class="menu__controls">
+                <button
+                type="button"
+                class="menu__button"
+                aria-expanded="false"
+                data-close-menu="close-menu"
+                >
+                  <svg
+                    class="icon icon__close menu__icon-close"
+                    width="24"
+                    height="24"
+                    aria-label="Закрити мобільне меню"
+                  >
+                    <use href="./images/icons/sprite.svg#icon-close"></use>
+                  </svg>
+                </button>
+                <ul class="nav menu__nav">
+                  <div class="nav__container">
+                    <li class="nav__item">
+                      <a href="#gallery" class="nav__link">Галерея</a>
+                    </li>
+                    <li class="nav__item">
+                      <a href="#events" class="nav__link">Виставки та події</a>
+                    </li>
+                  </div>
+                  <div class="nav__container">
+                    <li class="nav__item">
+                      <a href="#describe" class="nav__link">Про Нас</a>
+                    </li>
+                    <li class="nav__item">
+                      <a href="#contacts" class="nav__link">Контакти</a>
+                    </li>
+                  </div>
+                </ul>
+              </div>
+              <div class="schedule menu__schedule">
+                <div class="schedule__container">
+                  <p class="schedule__title">Графік роботи</p>
+                  <ul class="schedule__list">
+                    <li class="schedule__item">
+                      <span class="schedule__item--strong">12:00 — 20:00</span> середа
+                    </li>
+                    <li class="schedule__item">
+                      <span class="schedule__item--strong">12:00 — 20:00</span> четвер
+                    </li>
+                    <li class="schedule__item">
+                      <span class="schedule__item--strong">12:00 — 20:00</span> п’ятниця
+                    </li>
+                    <li class="schedule__item">
+                      <span class="schedule__item--strong">11:00 — 19:00</span> субота
+                    </li>
+                    <li class="schedule__item">
+                      <span class="schedule__item--strong">11:00 — 19:00</span> неділя
+                    </li>
+                  </ul>
+                </div>
+                <p class="schedule__weekend">
+                  <span class="schedule__weekend--strong">Вихідні:</span> понеділок, вівторок<br> Каса припиняє роботу за 1 годину до закриття музею
+                </p>
+              </div>
+            </div>
+            <div class="logo top-bar__logo">
+              <a href="#home">na<br>mu</a>
+            </div>
```

Comment:
I checked the site and I think it's unnecessary logic

Like the hover effect and animation super
![image](https://github.com/mate-academy/Museum/assets/97075591/6c7ce72e-d510-43b7-860c-c884bb048ee2)


But we don't need logic with the transition to home because we see it only when we are at the very top

It would be a cool solution if you had several pages and it would return us to the main page